models/user.go

```diff
@@ -1,7 +1,7 @@
 package models
 
 type User struct {
-	Id              int    `json:"-" gorm:"column:user_id"`
+	Id              string `json:"-" gorm:"column:user_id"`
```

Comment:
Tipo do ID no banco é uuid, está utilizando como string para facilitar o desenvolvimento?
Caso for utilizar uuid utilizar biblioteca do google: https://github.com/google/uuid

Reply:
Sim, não vou mudar pra uuid.